### PR TITLE
adding ability to use arrays for path

### PR DESCRIPTION
**Pull request by abritinthebay**
Addresses feature request raised in #3

### CODE REVIEW

**Comment by abritinthebay** [comment]
Will fix CI bugs later (perhaps tomorrow, it's late) and add some tests.

File: lib/index.js [hunk 1]
```diff
@@ -11,22 +11,31 @@ module.exports = (server, options, next) => {
...
+        // now we force path to be an Array (makes life easier later)
+        if (!Array.isArray(path)) {
+            path = [path];
+        }
 
         headers = headers || {};
 
-        // Adjust for relative path
-        path = (path[0] === '/') ? path : `${this.realm.modifiers.route.prefix || ''}/${path}`;
+        const pushes = path.map((path) => {
```
Comment (devinivy) [inline lib/index.js]: Mind using a for loop here?  I realize it's sort of a micro-optimization, but I try to keep close to hapi's best practices, which suggest using for loops in hot code paths like route handlers.  Rest assured I use `map()` elsewhere liberally!

**Comment by devinivy** [comment]
Cool!  Left a comment, but this is otherwise good to go.  Thanks for taking this up!

File: lib/index.js (hunk 1 above)
Comment (abritinthebay) [inline lib/index.js]: Sure! it makes my functional-programing-happy brain kinda sad but I totally get it 👍

**Comment by devinivy** [comment]
Hey @abritinthebay do you plan to take this over the finish line?  If not, no big deal– I'll be able to use your commits.  Just thinking about a release in the next week or two.

**Comment by abritinthebay** [comment]
Woah, brain fail on my part - sorry about that. Will take a look at doing that. Sorry - real life got in the way and a forgot totally >_<

**Comment by devinivy** [comment]
Not a worry! 👍 RL :1st_place_medal:

**Comment by underbyte** [comment]
Any chance of finishing this? I love to try it out soon.

**Comment by abritinthebay** [comment]
Something VERY wrong with the tests now. Will take some time to work out how the hell that broke so much. I switched it up to a for loop tho.

**Comment by abritinthebay** [comment]
Hmmm... the same tests fail when I checkout your current master and run them there. So I'll leave those test failures to you..?

**Comment by devinivy** [comment]
Very interesting– I'll take a peek.  I wonder if things have moved out from under us a bit, now that http2 has landed in node and there are potential conflicts with the userland http2 module.

This plugin is going to get a big refresher to work with node's http2 support before long :)

File: lib/index.js [hunk 2]
```diff
@@ -11,22 +11,32 @@ module.exports = (server, options, next) => {
...
+        }
 
         headers = headers || {};
 
-        // Adjust for relative path
-        path = (path[0] === '/') ? path : `${this.realm.modifiers.route.prefix || ''}/${path}`;
+        const pushes = [];
+        for (const currentPath of path) {
+            // Adjust for relative path
+            currentPath = (currentPath[0] === '/') ? currentPath : `${this.realm.modifiers.route.prefix || ''}/${currentPath}`;
```
Comment (devinivy) [inline lib/index.js]: The issue was this assignment to `currentPath`, which is a constant.  I'm working on this today, so I'm going to go ahead and fix it!

**Pull request merged**